dist/index.html

```diff
@@ -230,6 +249,7 @@ <h2>Loaded Chocolate Chip Pudding Cookie Cups</h2>
     </section>
     <section id="clickedRecipe" class="hidden">
       <section class="recipe-details" id="recipeLeftSide">
+        <section class="modal-add-recipe"></section>
```

Comment:
It looks like this is slightly deforming the absolute positioning of the image.  My thought is to maybe add this element as a child of `recipeName` and then mess around with some `absolute` positioning?  Any other thoughts?
<img width="723" alt="Screenshot 2023-05-20 at 4 44 51 PM" src="https://github.com/lauraguerra1/whats-cookin/assets/32660020/d482056b-dd06-47be-be9c-cab496d6cb9d">

Reply:
Another option is for us to add even more margin/padding to the ingredients i guess, which would be a quicker fix

Reply:
done